config/jobs/etcd/etcd-periodics.yaml

```diff
@@ -145,6 +147,53 @@ periodics:
     nodeSelector:
       kubernetes.io/arch: amd64
 
+- name: ci-etcd-robustness-arm64
+  interval: 24h
+  cluster: k8s-infra-prow-build
+  decorate: true
+  decoration_config:
+    timeout: 210m
+  extra_refs:
+    - org: etcd-io
+      repo: etcd
+      base_ref: main
+  annotations:
+    testgrid-create-test-group: 'true'
```

Comment:
Here is the source pr that introduced the feature: https://github.com/kubernetes/test-infra/pull/12476#discussion_r280952933

It's lacking on docs but there is some brief discussion of it.  Think it boils down to:

> testgrid-create-test-group boolean annotation, to force a test group to be created even if we aren't immediately adding it to a dashboard (because you configured that somewhere else)

This applies in our situation as we are defining dashboard elsewhere in separate file `config/testgrids/kubernetes/sig-etcd/config.yaml`.

Reply:
Got it. Thanks for the explanation.